src/main/java/frc/utils/calibration/swervecalibration/CharacterizationDirection.java

```diff
@@ -0,0 +1,17 @@
+package frc.utils.calibration.swervecalibration;
+
+public enum CharacterizationDirection {
+
+    CLOCKWISE(-1),
+    COUNTER_CLOCKWISE(1);
+
+    private final int directionSign;
+
+    CharacterizationDirection(int directionSign) {
+        this.directionSign = directionSign;
+    }
+
+    public int getDirectionSign() {
+        return directionSign;
+    }
```

Comment:
why not make the direction sign public and get rid of the getter?
you cannot change the direction sign because it is final so i see no reason not to